pkg/config/config_test.go

```diff
@@ -518,3 +697,49 @@ func createDirectoryAndFile(create bool, fs filesystem.Filesystem, odoDir string
 	}
 	return nil
 }
+
+func testDevfileObj(fs devfilefs.Filesystem) parser.DevfileObj {
```

Comment:
Please move this to the testingutil package

Reply:
updated